### DISCUSSION TITLE
Precise seeking for range components

### DISCUSSION THREAD

**Discussion by aryu-ki** [ppy/osu#27557] [Ideas]
Sometimes it's hard to pick specific value on the range, because pen would jump several ticks. It would be much easier to seek precisely by pulling range handle up, like in Youtube player
![image](https://github.com/ppy/osu/assets/84020506/9490662f-cb18-4f01-8387-c1b9d0fda3c9)

**Comment by Gabixel** [comment]
I'm not sure what you mean about the YouTube player, maybe the fact that you can skip a few seconds / a single frame, or that you can drag up the bar to better specify the time. If that's the case, I'm not sure if it fits with osu!'s design.

There's already a system of scrolling the sliders with the arrows on the keyboard, but obviously this is not the best option if you mainly use touch inputs.

To solve this, it's already planned that [a textbox to input precise values will be added](https://github.com/ppy/osu/discussions/23191#discussioncomment-5627061). Maybe this is not the best solution for pen users, but I hope it will still help!
See also:
- https://github.com/ppy/osu/discussions/24900
- https://github.com/ppy/osu/discussions/23191
- (I could swear there were others, but I can't find them)

**Reply by aryu-ki** [reply]
Well, there are lots of places where slider bars take place. E.g. setting difficulty range in song select might use the same enhancement. How does it not fit with osu! design if settings are filled with those slide bars as well? 
I'm not trying to insist too much, but for pen users it would be a dealbreaker really. I personally hate to readjust my pen grip, and I think most of the players struggle with consistency too and would find it to be a very cool improvement

**Reply by Gabixel** [reply]
Well, yeah, as a pen player myself, that could come in handy. It's just my thought at the moment (the fact that I don't know if it can work well with the current design), we'll have to see if it's something they want to implement or not.
I generally support this idea though!
